drake/examples/Quadrotor/runDynamics.cpp

```diff
@@ -1,25 +1,57 @@
 
 #include <iostream>
-#include "Quadrotor.h"
-#include "drake/systems/Simulation.h"
-#include "drake/systems/plants/BotVisualizer.h"
+#include "BotVisualizer.h"
```

Comment:
Got it.   I was following the car sim example.  Should I update that one as well?